shared/profile/search.js

```diff
@@ -0,0 +1,120 @@
+// @flow
+import React from 'react'
+import {onUserClick} from '../actions/profile'
+import * as Creators from '../actions/chat/creators'
+import * as SearchCreators from '../actions/searchv3/creators'
+import {debounce} from 'lodash'
+import {navigateUp} from '../actions/route-tree'
+import SearchHeader from '../chat/search-header'
+import ServiceFilter from '../searchv3/services-filter'
+import ResultsList from '../searchv3/results-list'
+import {compose, withState, withHandlers, defaultProps} from 'recompose'
+import {Box, Icon, Text} from '../common-adapters'
+import {connect} from 'react-redux'
+import {globalStyles, globalColors, globalMargins} from '../styles'
+
+import type {TypedState} from '../constants/reducer'
+
+type OwnProps = {}
+
+const mapStateToProps = ({chat: {searchResults}}: TypedState, {sidePanelOpen}: OwnProps) => ({
+  ids: searchResults.toArray(),
+})
+
+const mapDispatchToProps = (dispatch: Dispatch, {onBack, onToggleSidePanel}: OwnProps) => ({
+  _search: debounce(
+    (term: string, service) => dispatch(SearchCreators.search(term, 'chat:updateSearchResults', service)),
+    1e3
+  ),
+  _onClick: username => {
+    dispatch(navigateUp())
+    dispatch(onUserClick(username))
+  },
+  _clearSearchResults: id => dispatch(Creators.clearSearchResults()),
+  onShowTracker: id => console.log('show tracker of', id),
```

Comment:
Ah, this one doesn't get hooked up here -- the show tracker icon is only supposed to be visible when building a group, which we aren't here.